tools/common_lib/src/gemm.h

```diff
@@ -2245,10 +2244,11 @@ class GemmCmDispatcher : public GemmBaseDispatcher
     };
 
 public:
-    GemmCmDispatcher(create_params_t&& params, cm_params_t&& cm_params, IntelExtension& intc_ext, ID3D12Device* d3d12_device, IDMLDevice* dml_device, IDMLCommandRecorder* dml_cmd_recorder, ID3D12GraphicsCommandList* cmd_list)
+    GemmCmDispatcher(create_params_t&& params, cm_params_t&& cm_params, IntelExtension& intc_ext, ID3D12Device* d3d12_device, IDMLDevice* dml_device, IDMLCommandRecorder* dml_cmd_recorder, ID3D12GraphicsCommandList* cmd_list, bool use_stateless)
```

Comment:
use_stateless should be part of `cm_params_t` which is passed tho this ctor as "&& cm_params``

Reply:
Done.